src/main/java/uk/gov/companieshouse/efs/api/events/service/model/FesLoaderModel.java

```diff
@@ -12,23 +14,29 @@
     private String companyName;
     private String companyNumber;
     private String formType;
+    private boolean sameDay;
 
     public FesLoaderModel(String barcode, String companyName, String companyNumber, String formType,
```

Comment:
the PR description says that a builder has been added to this class.  Am I looking in the right place?

Reply:
Good spot, it should have said FormTemplate. I'll fix the description